tools/depends/target/Makefile

```diff
@@ -27,15 +27,16 @@ else
 endif
 
 ifeq ($(OS),ios)
-  EXCLUDED_DEPENDS = libcec libusb libdvdcss
+  EXCLUDED_DEPENDS = libcec libusb
   ifeq ($(TARGET_PLATFORM),appletvos)
     DEPENDS += boblight
   endif
   DEPENDS += iosentitlements
-else
-  DVDREAD_DEPS = libdvdcss
 endif
 
+DVDREAD_DEPS = libdvdcss
```

Comment:
as this variable now contains the same value for all possible configurations and only used once, it could be removed and replace with it's value